### ISSUE TITLE
Implement remaining unblocked ADC channels

### ISSUE THREAD

**Issue by zrnsm** [zrna-research/akso#3]
There are six ADC channels that should be relatively easy to add. Pins: C5, F6, F7, F8, F9, H4.

Changes needed at least in `axoloti_board.c` and in the factory analog in object.

**Comment by consumr** [comment]
I'd like to suggest that you further this exercise by opening all GPIO pins and possibilities in the factory objects. 
I know it's a big ask, but it's an enhancement that is certainly fundamental to the Akso's operation.

**Comment by zrnsm** [comment]
Hey @consumr . I agree. I think it's an open question as far as how to best expose the many IO possibilities of the H7 at the patcher object level. I'm not satisfied with how it's done in the legacy patcher (there are many things in the current patcher I'm not satisfied with but simply haven't had time to address yet).

In general, any pin that is not allocated for some core board function can be a simple digital IO. More complex functions are bound to particular pins, i.e. a particular physical pin has several possible functions. That mapping is defined in the H7 datasheet.

The issue is exposing that complexity in a way that is understandable to people who are not necessarily familiar with the low-level firmware details.

Which specific IO functions are highest priority from your perspective?

**Comment by shabazz10** [comment]
> There are six ADC channels that should be relatively easy to add. Pins: C5, F6, F7, F8, F9, H4.

Any updates on this? It's been quite a while now and this would be nice to have

**Comment by dbosnyak** [comment]
ok I will do a PR on this, I see how to change the objects themselves easily enough, will look into the changes to axoloti_board.c and test it out- my only real holdup is to get some pins soldered to the board and make a little proto db up.
